CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 * <How to hit and notice issue? what was the impact?> ([#????](https://github.com/realm/realm-core/issues/????), since v?.?.?)
 * Fix queries for null on non-nullable indexed integer columns returning results for zero entries. (Since v6)
```

Comment:
which v6?

Reply:
was this reported in a github issue?

Reply:
Is the changelog sentence crystal clear to everyone else but me? (same goes for the next changelog)

Reply:
@ironage  found these issues while improving some tests.